pygpcca/_gpcca.py

```diff
@@ -971,12 +975,20 @@ def optimize(
                     f"Can't check complex conjugate block splitting for {m} clusters with only "
                     f"{len(self._p_eigenvalues)} eigenvalues."
                 )
-            else:
-                if _check_conj_split(self._p_eigenvalues[:m]):
+            if _check_conj_split(self._p_eigenvalues[:m]):
+                if len(m_list) == 1:
```

Comment:
Do we always pass this as a list here?

Reply:
Yes, see the initialization part of `GPCCA.optimize`, e.g. [here](https://github.com/msmdev/pyGPCCA/blob/3c09ef64f6a687649adf586133acc44fc5682fdb/pygpcca/_gpcca.py#L915).